frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -440,6 +453,24 @@ export const managePipelineResources = async (
     );
   }
 
+  if (git.secret) {
+    const secret = await k8sGet(SecretModel, git.secret, project.name);
+    secret.metadata.annotations = _.merge(
+      secret.metadata.annotations,
+      getSecretAnnotations({ key: 'git', value: GitUrlParse(git.url).resource }),
```

Comment:
@rottencandy  based your screenshot,  I see you have used ssh url, but if you try with https url it will fail to pull from the repository because the annotation that is added here is always to support ssh url (git@github.com:karthikjeeyar/reactApp.git)
```
 tekton.dev/git-0: github.com
 ```
 we need also support the https urls as well, for https urls the annotation should be
 
 ```
 tekton.dev/git-0: https://github.com 
 ```

Reply:
Updated & added test.